ui/src/logs/utils/table.ts

```diff
@@ -157,3 +157,14 @@ export const applyChangesToTableData = (
     values: updatedValues,
   }
 }
+
+export const isEmptyInfiniteData = (data: {
```

Comment:
a little confused by this naming (`isEmptyInfiniteData`). does this replace the isTableEmpty() from above?

Reply:
Yeah, it's now a util that helps check when to set searchStatus to `LoadingStatus.NoResults` when the tableInfiniteData is empty.